bpaotu/bpaotu/templates/bpaotu/search.html

```diff
@@ -240,6 +240,9 @@ <h4>For instructions on how to use this search tool, please see <a href="/organi
                     <button type="button" class="btn btn-default " id="export_button">
                         <span class="glyphicon glyphicon-download" aria-hidden="true"></span> Export Search Results
                     </button>
+                    <button type="button" class="btn btn-default " id="export_biom_button">
+                        <span class="glyphicon glyphicon-download" aria-hidden="true"></span> Export Search Results (biom format)
```

Comment:
change the first button to export... (CSV) and the other to export... (BIOM)

Reply:
Fixed.